internal/common/metrics/constants.go

```diff
@@ -74,3 +74,23 @@ const (
 
 	WorkflowActiveThreadCount = TemporalMetricsPrefix + "workflow_active_thread_count"
 )
+
+// Metric tag keys
+const (
+	NamespaceTagName        = "namespace"
+	ClientTagName           = "temporal_client_name"
```

Comment:
```suggestion
	ClientTagName           = "client_name"
```
I renamed header name also.